reconcile/glitchtip/reconciler.py

```diff
@@ -0,0 +1,278 @@
+import logging
+from typing import Iterable, Sequence
+
+from reconcile.utils.glitchtip import GlitchtipClient, Organization, Project, Team, User
+
+
+class GlitchtipReconciler:
```

Comment:
Why is this a class?

Reply:
What do you mean?

Reply:
Yes, you're right; it can be both. I prefer classes over methods :)